benchmarks/cluster/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+add_library(cluster_benchmark_helpers
+  data.cpp
+  print_timers.cpp
+)
+target_compile_features(cluster_benchmark_helpers PRIVATE cxx_std_17)
+target_link_libraries(cluster_benchmark_helpers PRIVATE ArborX::ArborX)
+
+set(input_file "input.txt")
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/${input_file} ${CMAKE_CURRENT_BINARY_DIR}/${input_file} COPYONLY)
+
+add_executable(ArborX_Benchmark_DBSCAN.exe dbscan.cpp)
+target_include_directories(ArborX_Benchmark_DBSCAN.exe PRIVATE ${CMAKE_CURRENT_SOURCE_DIR})
+target_link_libraries(ArborX_Benchmark_DBSCAN.exe ArborX::ArborX Boost::program_options cluster_benchmark_helpers)
```

Comment:
```suggestion
target_link_libraries(ArborX_Benchmark_DBSCAN.exe PRIVATE ArborX::ArborX Boost::program_options cluster_benchmark_helpers)
```

Reply:
Doesn't really matter for the executable. And we don't do this anywhere else.